src/api/liveSpecsExt.ts

```diff
@@ -339,6 +339,46 @@ const getLiveSpecsByCatalogNames = async (
     );
 };
 
+export interface LiveSpecsExtQuery_GroupedUpdates {
+    catalog_name: string;
+    id: string;
+}
```

Comment:
This interface should be defined in the _types.ts_ file in the _src/api_ directory.